mmdet/models/dense_heads/deformable_detr_head.py

```diff
@@ -96,75 +74,41 @@ def init_weights(self) -> None:
+            init_reference (Tensor): The initial reference points entered
+                into the decoder, has shape [bs, num_query, 4].
+            inter_references (Tensor): The intermediate reference points
+                in each decoder layer, has shape
+                [num_decoder_layers, num_query, bs, 4].
 
         Returns:
-            tuple[Tensor]:
+            tuple[Tensor]: results of decoder containing the following tensor.
 
-            - all_cls_scores (Tensor): Outputs from the classification head,
+            - outputs_classes (Tensor): Outputs from the classification head,
               shape [nb_dec, bs, num_query, cls_out_channels].
-            - cls_out_channels should includes background.
-            - all_bbox_preds (Tensor): Sigmoid outputs from the regression
+            - outputs_coords (Tensor): Sigmoid outputs from the regression
               head with normalized coordinate format (cx, cy, w, h).
               Shape [nb_dec, bs, num_query, 4].
-            - enc_outputs_class (Tensor): The score of each point on encode
-              feature map, has shape (N, h*w, num_class). Only when
-              as_two_stage is True it would be returned, otherwise `None`
-              would be returned.
-            - enc_outputs_coord (Tensor): The proposal generate from the
-              encode feature map, has shape (N, h*w, 4). Only when
-              as_two_stage is True it would be returned, otherwise `None`
-              would be returned.
         """
-
-        batch_size = x[0].size(0)
-        input_img_h, input_img_w = batch_img_metas[0]['batch_input_shape']
-        img_masks = x[0].new_ones((batch_size, input_img_h, input_img_w))
-        for img_id in range(batch_size):
-            img_h, img_w = batch_img_metas[img_id]['img_shape']
-            img_masks[img_id, :img_h, :img_w] = 0
-
-        mlvl_masks = []
-        mlvl_positional_encodings = []
-        for feat in x:
-            mlvl_masks.append(
-                F.interpolate(img_masks[None],
-                              size=feat.shape[-2:]).to(torch.bool).squeeze(0))
-            mlvl_positional_encodings.append(
-                self.positional_encoding(mlvl_masks[-1]))
-
-        query_embeds = None
-        if not self.as_two_stage:
-            query_embeds = self.query_embedding.weight
-        hs, init_reference, inter_references, \
-            enc_outputs_class, enc_outputs_coord = self.transformer(
-                    x,
-                    mlvl_masks,
-                    query_embeds,
-                    mlvl_positional_encodings,
-                    reg_branches=self.reg_branches if self.with_box_refine else None,  # noqa:E501
-                    cls_branches=self.cls_branches if self.as_two_stage else None  # noqa:E501
-            )
-        hs = hs.permute(0, 2, 1, 3)
+        hidden_states = hidden_states.permute(0, 2, 1, 3)
         outputs_classes = []
```

Comment:
all_layers_outputs_classes and all_layers_outputs_coords

Reply:
Hi~ I have not understand the mearning of this comment ~

Reply:
Did you mean that 'rename the output args name of `bbox_head.forward()`: 
from `outputs_classes`, `outputs_coords` to `all_layers_outputs_classes ` and `all_layers_outputs_coords`' ?

Reply:
yes